integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/AlibabaDubboTransactionPropagationFilter.java

```diff
@@ -43,45 +45,51 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
             return invoker.invoke(invocation);
         }
         String xid = RootContext.getXID();
-        String xidInterceptorType = RootContext.getXIDInterceptorType();
+        String branchType = RootContext.getBranchType();
 
         String rpcXid = getRpcXid();
-        String rpcXidInterceptorType = RpcContext.getContext().getAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE);
+        String rpcBranchType = RpcContext.getContext().getAttachment(RootContext.KEY_BRANCH_TYPE);
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("xid in RootContext[{}] xid in RpcContext[{}]", xid, rpcXid);
         }
         boolean bind = false;
         if (xid != null) {
             RpcContext.getContext().setAttachment(RootContext.KEY_XID, xid);
-            RpcContext.getContext().setAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE, xidInterceptorType);
+            RpcContext.getContext().setAttachment(RootContext.KEY_BRANCH_TYPE, branchType);
         } else {
             if (rpcXid != null) {
                 RootContext.bind(rpcXid);
-                RootContext.bindInterceptorType(rpcXidInterceptorType);
+                if (StringUtils.equals(BranchType.TCC.name(), rpcBranchType)) {
+                    RootContext.bindBranchType(BranchType.TCC);
+                }
                 bind = true;
                 if (LOGGER.isDebugEnabled()) {
-                    LOGGER.debug("bind[{}] interceptorType[{}] to RootContext", rpcXid, rpcXidInterceptorType);
+                    LOGGER.debug("bind xid [{}] branchType [{}] to RootContext", rpcXid, rpcBranchType);
                 }
             }
         }
         try {
             return invoker.invoke(invocation);
-
         } finally {
             if (bind) {
-                String unbindInterceptorType = RootContext.unbindInterceptorType();
                 String unbindXid = RootContext.unbind();
+                String previousBranchType = RootContext.getBranchType();
+                if (StringUtils.equals(BranchType.TCC.name(), previousBranchType)) {
+                    RootContext.unbindBranchType();
+                }
                 if (LOGGER.isDebugEnabled()) {
-                    LOGGER.debug("unbind[{}] interceptorType[{}] from RootContext", unbindXid, unbindInterceptorType);
+                    LOGGER.debug("unbind xid [{}] branchType [{}] from RootContext", unbindXid, previousBranchType);
```

Comment:
BranchType print null in AT mode is not very elegant.

Reply:
How about judge in the log output temporarily,and print "AT" if it is null